spec/spec_helper_local.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'pathname'
+dir = Pathname.new(__FILE__).parent
+$LOAD_PATH.unshift(dir, File.join(dir, 'fixtures/modules/augeasproviders_core/spec/lib'), File.join(dir, '..', 'lib'))
+
+require 'augeas_spec'
+
+# augeasproviders: setting $LOAD_PATH to work around broken type autoloading'
```

Comment:
Have you verified this is still needed? Also, it's weird that you have mostly the same code on line 5. Does `Puppet.initialize_settings` remove it from `$LOAD_PATH`?

Reply:
cleaned it up